src/components/CloseButton/style.css

```diff
@@ -0,0 +1,31 @@
+.close-button {
```

Comment:
component style files are usually called like a component `CloseButton.css`

Reply:
Thank you for your note. I will remember this note and I will rename all CSS files in the next PR.